aries-site/src/pages/components/header.mdx

```diff
@@ -20,127 +23,91 @@ import {
   <HeaderExample />
 </Example>
 
-## Guidance
-
-Header is used to set the context for an application or a specific
-page within an application. It is a flexible element that can be
-composed of a variety of components to aid the user experience in an
-application.
+## When to use
 
-### About Header
-
-Header is extremely versatile and can be built by mixing and matching a variety of components to provide users with relevant interactive elements to navigate through or engage with an application.
+Header is used to set the context for an application or a specific page within an application. It is a flexible element that can be composed of a variety of components to aid the user experience in an application.
 
 **Common use cases include:**
 
-- Creating a navigation structure for an application
-- Providing a search field for an application or single page within an application
-- Displaying an action button that affects the application or current page content
-
-**Components used within a Header:**
+- [Global header](#global-header)
+- [App header](#app-header)
+- [Page header](#page-header)
 
-Some of the most common components that can be used in Header are [Button](/components/button), Avatar, [Search](/components/search), and [Menu](/components/menu).
+## When not to use
 
-### Usage
+Do not use Header just as a means to display content in a row. The Header component translates to a `<header>` in the DOM, so do not use Header in instances that do not align with the [HTML header spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).
 
-While Header is flexible in its composition, there are some important guidelines and use cases to consider. Following these examples and guidance will help ensure your Header is compliant with the HPE Design System standards.
+## Anatomy
 
-**Keep Header contents to a limited set of elements in order to:**
+While Header is flexible in its composition, these anatomical elements are the most common:
 
-- Provide the user with relevant, helpful actions regarding the application or page
-- Layout smoothly and responsively across all browser widths
+- [Button](/components/button)
+- [Menu](/components/menu)
+- [Search](/components/search)
+- [Heading](/foundation/typography#heading)
+- Avatar
```

Comment:
Should "logo" also be listed? or do you consider that part of "Avatar" ?

Reply:
Good question, the logo can be grouped under Avatar for non-Global Header cases. Since the global header logo is unique, I'll just leave that for the global header section.